src/UIAutomation/UiaOperationAbstraction/UiaOperationAbstraction.h

```diff
@@ -123,6 +123,55 @@ namespace UiaOperationAbstraction
             }
         }
 
+        template<class TryBody, class CatchBody>

```

Comment:
Could we add tests that validate try-catch behavior in the local and remote cases?